model/wrapper.js

```diff
@@ -0,0 +1,44 @@
+var bcrypt = require('bcrypt')
+const saltRounds = 10
+
+function createUser(connection, uname, email, password, admin) {
+    bcrypt.hash(password, saltRounds, function (err, hash) {
+        var query = 'INSERT INTO users VALUES(' + email + ', ' + hash + ', ' + uname +  ', ' + admin + ')'
+        connection.query(query, function (err, rows, fields) {
+            if (err) {
+                throw "Failed to execute create user SQL"
+            } else {
+                return true
+            }
+        })
+    })
+}
+
+function userExists(connection, email) {
+    var query = 'SELECT uname FROM users WHERE email = ' + email
+    connection.query(query, function (err, rows, fields) {
+        if (err) {
+            throw "Failed to execute user exists SQL"
+        }
+        return (rows.count > 0)
+    })
+}
+
+function shouldLogin(connection, email, password) {
```

Comment:
This name is weird, maybe something like "checkCredentials" or "verifyCredentials" is more appropriate?